design/ceph/object/store.md

```diff
@@ -70,6 +72,11 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
     erasureCoded:
       dataChunks: 6
       codingChunks: 2
+  radosNamespaces:
+    # If true, create RADOS namespaces for this object store.
+    # Default is false, preserve the current behavior for backward compatibility.
+    create: true
+    poolName: my-pool-collection
```

Comment:
```suggestion
    poolName: rgw-common-pool
```

Reply:
How would they create this pool? With a CephBlockPool CR? IIRC any pools created with that CR will be marked as the `rbd` application, but would the pool need to set the `rgw` application instead?

Reply:
IMO it is better to leave to users about creating the pools, It may add up complexity about which store is owner and when to delete and all.